lib/src/equatable_utils.dart

```diff
@@ -7,7 +7,12 @@ int mapPropsToHashCode(Iterable<Object?>? props) {
 }
 
 /// Determines whether two iterables are equal.
+@pragma('vm:prefer-inline')
 bool iterableEquals(Iterable<Object?> a, Iterable<Object?> b) {
+  assert(
```

Comment:
Thoughts on calling `setEquals` instead of asserting here?

```dart
if (a is Set && b is Set) return setEquals(a, b);
```

Reply:
I don't expect anyone to use this function. I'm not sure it's even exported. So I'd prefer to leave it simple, without extra ifs